backend/src/email/services/email.service.ts

```diff
@@ -156,7 +156,15 @@ const sendEmail = async (
   opts?: SendEmailOpts
 ): Promise<boolean> => {
   try {
-    await MailService.mailClient.sendMail(mail, opts)
+    const channelProviderMessageId = await MailService.mailClient.sendMail(
```

Comment:
I'm guessing it's intentionally vague? i.e. you didn't want to specify it's AWS SES? not sure if we want to add comments here so it's clearer, or is the idea that you want the code to be agnostic of the actual provider. 

maybe `emailServiceProvider` might be clearer?

Reply:
Yeah, the name of the module (`email.service.ts`) is non-AWS specific so I felt variable names should reflect this. Although I do prefer your suggestion of at least being concrete about the channel (i.e. `emailProviderMessageId`). Will make that change.